packages/feature-toggles/src/resources/__tests__/EvidentlyFeatureRepository.spec.ts

```diff
@@ -0,0 +1,56 @@
+import Evidently, { ClientConfiguration } from 'aws-sdk/clients/evidently';
+import EvidentlyFeatureRepository from '../EvidentlyFeatureRepository';
+
+jest.mock('aws-sdk/clients/evidently');
+
+describe('EvidentlyFeatureRepository', () => {
+    it('should evaluate a feature for a given entity', async () => {
+        const expectedResult = { variation: 'T1', value: 'DummyValue' };
+
+        const evaluateMock = jest.fn(() => Promise.resolve(expectedResult));
+        (Evidently as any).mockImplementation(() => ({
```

Comment:
The change includes many `any`s in the tests. While it's relatively harmless, it's possible to lie in the tests on accident. Is it possible to cast as the correct type?

Reply:
I did one better, I updated these and flipped the eslint rule from warning to error.